ui/src/components/retreat/stream/StreamContainer.js

```diff
@@ -0,0 +1,106 @@
+import {
+  VStack,
+  Box,
+  Button,
+  Stack,
+  AspectRatio,
+  Link,
+  Text,
+  Container,
+  Flex
+} from '@chakra-ui/react';
+import { ChevronLeftIcon } from '@chakra-ui/icons';
+import OnlinePageButtons from './StreamButtons';
+import { customAxios as axios } from '../../helpers/customAxios';
+import React, { useState, useEffect, useCallback } from 'react';
+import '@fontsource/sora';
+import '@fontsource/inter';
+
+const StreamContainer = (props) => {
+  const [title, setTitle] = useState("With Everything Congregational Retreat 2022")
+  const [url, setUrl] = useState("");
+
+  // TODO-aparedan: test with real data
+  const getData = useCallback(async () => {
+    try {
+      const { data, status } = await axios.get('/api/sermons/get-sermons', {
+        params: {
+          includeProtected: true,
+          includePublic: false
+        }
+      });
+      if (status === 200 && data[0]) {
+        setTitle(data[0].title);
+        setUrl(data[0].streamLink);
+      }
+    } catch (err) {
+      console.log(err);
+    }
+  }, []);
+
+  useEffect(() => {
+    (async () => {
+      await getData();
+    })();
+  }, [getData]);
+
+  const OnlineSection = () => {
+    return (
+      <>
+        <OnlinePageButtons />
+      </>
+    );
+  };
+
+  return (
+    <Flex
+      w="full"
+      h="100vh"
```

Comment:
i think it's better to not hardcode the height like this, on my computer the buttons are overflowing. maybe a `minH` would be better instead of a hardcoded h